apps/src/blockly/customBlocks/googleBlockly/mutators/functionMutatorHelpers.js

```diff
@@ -23,9 +23,9 @@ export function setBlockDescription(block, extraState) {
 // from the description field on the block. Otherwise we get it from
 // the description property on the block.
 export function getBlockDescription(block) {
-  let description = block.description;
+  let fieldDescription;
   if (Blockly.useModalFunctionEditor) {
-    description = block.getFieldValue('DESCRIPTION');
+    fieldDescription = block.getFieldValue('DESCRIPTION');
   }
-  return description;
+  return fieldDescription || block.description;
```

Comment:
This change is unrelated to the PR but was committed. It ensures that description fields are properly populated for shared behaviors. These blocks have a description property but do not automatically have a value set in the new description field on the block.